util/data.ts

```diff
@@ -22,6 +23,15 @@ export const normalizeKeys = object => {
     }, {});
 };
 
+export const matchCountryCode = update => {
+  const countryName = update.countryRegion;
+  const countryCode = Object.entries(countries).filter(
```

Comment:
Better to use `find` method. At the line 31, probably can be breaked if an empty array was returned

Don't forget to check again, because your countries data not 100% match with previous data. eg: `US` is also a country name previously.

Reply:
Thanks I'll give that a shot later. Any idea how to get to the nested array value?

Reply:
I don't have more idea since your country definitions using `code` as the `key`. Maybe we can switch it? You can use country name as key and code as value. 

Then, you no need to loop it again with `find` or `filter`. Just match it with `countries[name]` and then you got the `code`

Here is the example snippet code:
```
const countryMap = {
      'Mainland China': 'CHN',
      'Singapore': 'SGP',
      'Indonesia': 'IDN',
      ...etc
}

(await fetchFeatures(endpoints.confirmedDesc))
      .map(attributeSpreader)
      .map(normalizeKeys)
      .map((x) => {
              //log if not found
              x.countryCode = countryMap[x. countryRegion];
              return x;
       })

```

Reply:
Yeah thats how I pictured it initally as well, but then I got the data from a http://country.io/data/ like that. I'll swap the key/values though as that way seems much easier.